draft-brunstrom-taps-impl.md

```diff
@@ -328,6 +328,9 @@ Branching for derived endpoints is the final step, and may have multiple layers
 Implementations should sort the branches of the tree of connection options in order of their preference rank. 
 Leaf nodes on branches with higher rankings represent connection attempts that will be raced first.
 Implementations should order the branches to reflect the preferences expressed by the application for its new connection, including Protocol and Path Selection Properties, which are specified in {{I-D.trammell-taps-interface}}. 
+Path Selection Properties take precedence over Protocol Selection Properties.
+
+If Protocol and Path Selection Properties contain any prohibited properties, the implementation should purge branches containing nodes with these properties.
```

Comment:
Does this mean that Prohibit takes precedence over all others? (I think it does, and I think it's correct). I think what I'd like to see here (and maybe I'm just a languages geek) is a strict ordering, e.g. something like:

`Prohibit` > `Require` > `Prefer` > `Avoid`

Reply:
Makes sense. I added more text to the implementation that details how this should work.